src/lib/bootloader/grub_install.rb

```diff
@@ -90,6 +94,26 @@ def basic_cmd(secure_boot, trusted_boot)
       cmd
     end
 
+    # Fixup for SecureBoot without shim, on aarch64 (boo#1136601)
+    # FIXME: remove me when/if 'grub2-install' command supports secureboot
+    def fixup_no_shim_secureboot(secure_boot, trusted_boot)
+      # Fixup only for secureboot on aarch64
+      return if !secure_boot || !Yast::Arch.aarch64
+      if trusted_boot
+        cmd = ["cp", "/usr/share/grub2/arm64-efi/grub-tpm.efi", "/boot/efi/EFI/*suse/bootaa64.efi"]
+      else
+        cmd = ["cp", "/usr/share/grub2/arm64-efi/grub.efi", "/boot/efi/EFI/*suse/bootaa64.efi"]
```

Comment:
does this *suse works? Execute does not use shell for expansion of path.

Reply:
I do not know, I am currently trying to get some review of the idea before trying it.
Any idea how to get the folder name for `/boot/efi/EFI/` ? It should be suse or opensuse, depending on the flavor.

Reply:
let me show example code in irb ( irb is your ruby friend ). I create /tmp/suse and /tmp/opensuse.
```
irb(main):001:0> Dir["/tmp/*suse"]
=> ["/tmp/opensuse", "/tmp/suse"]
```
so `Dir[]` with glob returns array of matched entries. If there is just one, then I suggest to use `Dir["/boot/efi/EFI/*suse"].first`.

Reply:
With my latest update, I just make use of `--suse-signed-grub` option for `grub2-mkimage`.